ams/server/src/main/java/com/netease/arctic/server/persistence/SqlSessionFactoryProvider.java

```diff
@@ -84,7 +86,7 @@ public void init(Configurations config) {
     dataSource.setNumTestsPerEvictionRun(BaseObjectPoolConfig.DEFAULT_NUM_TESTS_PER_EVICTION_RUN);
     dataSource.setTestOnReturn(BaseObjectPoolConfig.DEFAULT_TEST_ON_RETURN);
     dataSource.setSoftMinEvictableIdleTimeMillis(
-        BaseObjectPoolConfig.DEFAULT_SOFT_MIN_EVICTABLE_IDLE_TIME.toMillis());
+            BaseObjectPoolConfig.DEFAULT_SOFT_MIN_EVICTABLE_IDLE_TIME.toMillis());
```

Comment:
@Kyofin Amoro project uses 2 spaces for indent and tab and 4 spaces for continuation indent for java codes.

Could you please change the configuration and reformat the codes?